pkg/http/api.go

```diff
@@ -476,13 +476,13 @@ func (a *api) onDeleteState(reqCtx *fasthttp.RequestCtx) {
 		log.Debug(msg)
 		return
 	}
-	respondEmpty(reqCtx, fasthttp.StatusOK)
+	respondEmpty(reqCtx, fasthttp.StatusNoContent)
 }
 
 func (a *api) onGetSecret(reqCtx *fasthttp.RequestCtx) {
 	if a.secretStores == nil || len(a.secretStores) == 0 {
 		msg := NewErrorResponse("ERR_SECRET_STORE_NOT_CONFIGURED", messages.ErrSecretStoreNotConfigured)
```

Comment:
`ERR_SECRET_STORES_NOT_CONFIGURED` ?  consistent with `ERR_STATE_STORES_NOT_CONFIGURED` or have singular in both?

Reply:
Good question, can you have more than one secret store? I don't actually know?

Reply:
Yes, you can.

Reply:
Then plural it is.